stories/report-error.stories.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * ReportError stories.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import Modules from 'googlesitekit-modules';
+import {
+	createTestRegistry,
+	WithTestRegistry,
+} from '../tests/js/utils';
+
+import ReportError from '../assets/js/components/ReportError';
+
+const { createModuleStore } = Modules;
+
+const error = {
+	code: 'missing_required_param',
+	message: 'Request parameter is empty: metrics.',
+	data: {},
+};
+
+storiesOf( 'Global', module )
+	.addDecorator( ( storyFn ) => {
+		const registry = createTestRegistry();
+		const testModuleDefinition = createModuleStore( 'test-module' );
+		registry.registerStore( testModuleDefinition.STORE_NAME, testModuleDefinition );
+		registry.dispatch( 'core/modules' ).registerModule( 'test-module', { name: 'Test Module' } );
```

Comment:
Let's import the store name as we would normally.
```suggestion
		registry.dispatch( CORE_MODULES ).registerModule( 'test-module', { name: 'Test Module' } );
```